src/axom/quest/examples/quest_distributed_distance_query_example.cpp

```diff
@@ -57,6 +57,8 @@ struct Input
 {
 public:
   std::string meshFile;
+  std::string distanceFile {"closest_point"};
```

Comment:
I borrowed the changes in this file from #879 and attributed the commit to @gunney1